src/xunit.console.netcore/Program.cs

```diff
@@ -20,7 +20,7 @@ public static int Main(string[] args)
         {
             try
             {
-                SetConsoleForegroundColor(ConsoleColor.White);
```

Comment:
These changes, while harmless, were no longer needed with newer versions of System.Console thanks to @ianhays